src/quemb/molbe/autofrag.py

```diff
@@ -1,8 +1,12 @@
 # Author: Oinam Romesh Meitei, Shaun Weatherly
 
+import re
 from copy import deepcopy
+from pathlib import Path
 from typing import Sequence
```

Comment:
Sequence should preferably imported from `collections.abc`.
https://docs.python.org/3/library/typing.html#typing.Sequence